src/components/utils/renderToContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import { ReactElement } from 'react';
+
```

Comment:
Can you add a heading comment that this can be removed when we stop supporting react 17

Reply:
updated!